tests/python/pytorch/nn/test_nn.py

```diff
@@ -1788,6 +1788,58 @@ def forward(self, graph, feat):
     explainer.explain_graph(g, feat, target_class=0)
 
 
+@parametrize_idtype
+@pytest.mark.parametrize(
+    "g",
+    get_cases(
+        ["homo"],
+        exclude=[
+            "zero-degree",
+            "homo-zero-degree",
+            "has_feature",
+            "has_scalar_e_feature",
+            "row_sorted",
+            "col_sorted",
+            "batched",
```

Comment:
Allow testing a batched graph as well?

Reply:
Ohh, yes, it makes sense with the updated implementation. I have removed it from the excluded list.